Apps/Database/src/Models/MessagingVerification.cs

```diff
@@ -72,6 +72,7 @@ public class MessagingVerification : AuditableEntity
         /// Gets or sets the SMS number for this verification.
         /// Required if the VerificationType = MessagingVerificationType.SMS.
         /// </summary>
+        [MaxLength(10)]
```

Comment:
could it be 11 characters when counting the country code?

Reply:
We don't use the country code and that would be a bigger change as UserProfile would have to be updated to match.